contracts/util/SettingsConsts.sol

```diff
@@ -78,4 +78,9 @@ contract SettingsConsts {
      */
     bytes32
         public constant OVER_COLLATERALIZED_BUFFER_SETTING = "OverCollateralizedBuffer";
+    /**
+        @notice The setting name for the maximum total value locked.
+        @notice The maximum total value locked is the supply cap for protocol, above which no additional deposits will be accepted.
+     */
+    bytes32 public constant MAXIMUM_TOTAL_VALUE_LOCKED = "MaximumTotalValueLocked";
```

Comment:
probably if we make it explicitly related to the pool would be easier to understand, something like: -POOL_MAXIMUM_TOTAL_VALUE_LOCKED instead of MAXIMUM_TOTAL_VALUE_LOCKED

Reply:
Changed